src/app/organization/organization.module.ts

```diff
@@ -36,7 +38,9 @@ import { ViewEmployeeComponent } from './employees/view-employee/view-employee.c
   imports: [
     SharedModule,
     OrganizationRoutingModule,
-    PipesModule
+    PipesModule,
+    MatCardModule,
```

Comment:
shared module already has angular material imports why is there need to seperately import them?

Reply:
In this branch pipe module has not been imported in shared module

Reply:
pipes module is not intended to be a part of shared module, remove angular material imports